### DISCUSSION TITLE
Very good niche imo

### DISCUSSION THREAD

**Discussion by nhh** [c3lang/c3c#241] [General]
Hi 🙋‍♂️ 

I just want to say, that i like the idea of c3 being a niche product between c and c++.

C always felt tedious and inconveniant. C++ feels incredibly heavy and has so much "big ideas".

Would love to see this become usable in the next years 💪

Coming from very high level languages, I dont really know if i can contribute something meaningful code-wiss, but I think I have a good gut-feeling of language design and language features.

Maybe I will contribute this by toying around with c3 and give you feedback :)

**Comment by lerno** [comment]
Feedback is always very welcome!
